torch/csrc/jit/passes/fold_conv_bn.cpp

```diff
@@ -353,12 +346,32 @@ void FoldConvBatchNorm2dHelper::transform() {
 
 } // namespace
 
-Module FoldConvBatchNorm2d(const Module& module) {
-  FoldConvBatchNorm2dHelper h;
+Module FoldConvBatchNorm(const Module& module) {
+  FoldConvBatchNormHelper h1, h2;
   Module m = module.clone();
-  addBiasForConv2dIfNone(m);
-  h.analyze(m);
-  h.transform();
+
+  addBiasForConvIfNone(m, "Conv2d");
+  addBiasForConvIfNone(m, "Conv3d");
+  // Conv2d + BatchNorm2d
+  const PatternInfo pattern2d = PatternInfo::parse_from_str(R"IR(
+graph(%self, %x):
+    %conv_submodule = match::module[name=".Conv2d"](%self)
```

Comment:
we can now match more exactly, please take a look at https://codebrowser.bddppq.com/pytorch/pytorch/torch/csrc/jit/passes/quantization/insert_observers.cpp.html#507